server/offenceCodeDecisions/Answer.ts

```diff
@@ -0,0 +1,155 @@
+import { getProcessedText, PlaceholderValues } from './Placeholder'
```

Comment:
We now have a Question(called decision) / Answer structure because the old way was all the wrong way around and I couldn't see a way to make it not counter intuitive